e2e/stateful-pallet-storage/handlePaginated.test.ts

```diff
@@ -142,7 +143,7 @@ describe('📗 Stateful Pallet Storage', function () {
       const payload_1 = new Bytes(ExtrinsicHelper.api.registry, 'Hello World From Frequency');
       const fake_schema_id = new u16(ExtrinsicHelper.api.registry, badSchemaId);
       const paginated_add_result_1 = ExtrinsicHelper.upsertPage(
-        providerKeys,
+        delegatorKeys,
```

Comment:
Same here as with `e2e/stateful-pallet-storage/handleItemized.test.ts` Perhaps we should revert to provider here if the delegator calls are covered by `e2e/stateful-pallet-storage/handleSignatureRequired.test.ts`?

Or do we need to duplicate these?

Reply:
The itemized and paginated extrinsics are handled in 2 separate files. Similar to the other test referenced, I had to change the caller here because the order of tests in the extrinsic has changed--this test would always have failed for an invalid delegate, but that was not the error thrown because it was checking the schemaId first; now it checks the delegation first and was throwing a different error, so I tweaked the test to pass for the delegate so we could get the desired `InvalidSchemaId` error.